docs/walkthrough/finetuning-models.md

```diff
@@ -1,10 +1,128 @@
 ---
-sidebar_position: 2
+sidebar_position: 4
 ---
 
 # Finetuning Models
 
 <!-- THE MARKDOWN (.md) FILE IS GENERATED FROM THE NOTEBOOK (.ipynb) FILE -->
+The large pretrained base models are already very helpful, but
+with finetuning smal and large language models can be made even more helpful.
 
-Finetuning a model...
+Finetunig is especially helpful for use cases that require private internal
+data or for use cases that require domain specific knowledge that wasn't
+available on the internet.
 
+In Substratus the Model resource allows you to specify a `baseModel` and
+a `trainingDataset` to make it very easy to create new finetuned models.
+
+To summarize: 
+`finetunedModel = training_image(baseModel, trainingDataset)`. 
+
+
+The following Model resource would create a finetuned model:
+```yaml
+apiVersion: substratus.ai/v1
+kind: Model
+metadata:
+  name: falcon-7b-instruct-k8s
+spec:
+  image:
+    name: substratusai/model-trainer-huggingface
+  baseModel:
+    name: falcon-7b-instruct
```

Comment:
The guide should mention that this model must have also been imported by a huggingface importer. It cant be any model.

Reply:
It's mentioned below but also added a note directly in the YAML